runtime/compiler/env/J9ClassEnv.hpp

```diff
@@ -145,7 +144,7 @@ class OMR_EXTENSIBLE ClassEnv : public OMR::ClassEnvConnector
     * \brief
     *    Checks whether instances of the specified class can be trivially initialized by
     *    "zeroing" their fields.
-    *    In the case of OpenJ9, this tests whether any field is of a primitive value type that
+    *    In the case of OpenJ9, this tests whether any field is of null-restricted field that
```

Comment:
Typo:  "any field is of null-restricted field" - I think that should read "any field is of a null-restricted type" or something like that.

Reply:
Addressed in [ab38690](https://github.com/eclipse-openj9/openj9/compare/7bc96d56b54a0663b220ede073dc58ef195f2dbe..ab3869094902f9ea93a5eb7b83504176ea6ee906)